City-of-Helsinki/parking-permits-ui
# Webshop: Add checkout-link to frontpage

## Description

Adds checkout URLs to notification on user landing page if any payments pending.

## Context

This is required so that if the user accidentally closes their browser tab in the middle of the payment process, or for some other reason, is unable to complete payment for an order.

In that case the "Payment pending" notification should include a link to the Talpa checkout page for each relevant pending order, so the user can try to complete the order.

[PV-612](https://helsinkisolutionoffice.atlassian.net/browse/PV-612)

## How Has This Been Tested?

Tested manually

## Manual Testing Instructions for Reviewers

NOTE: this requires that `checkout_url` is available in GraphQL returned from server, see https://github.com/City-of-Helsinki/parking-permits/pull/378

1. As a Webshop user with no current permits, create a new permit.
2. When you reach the Talpa checkout page, close the browser tab/window.
3. Go back to the Webshop landing page. You should see a screen similar to the one in the screenshot.
4. Clicking on the "Complete order" link should take you back to the Talpa checkout page, so you can view and complete the order.

## Screenshots

![Screenshot from 2023-09-11 14-46-06](https://github.com/City-of-Helsinki/parking-permits-ui/assets/131681805/8bbf9928-a98b-4b10-a3a4-46a713671a97)



[PV-612]: https://helsinkisolutionoffice.atlassian.net/browse/PV-612?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
